utils/_context/_scenarios/__init__.py

```diff
@@ -601,6 +601,22 @@ def all_endtoend_scenarios(test_object):
         scenario_groups=[ScenarioGroup.DEBUGGER],
     )
 
+    debugger_symdb = EndToEndScenario(
+        "DEBUGGER_SYMDB",
+        rc_api_enabled=True,
+        weblog_env={
+            "DD_DYNAMIC_INSTRUMENTATION_ENABLED": "1",
+            "DD_SYMBOL_DATABASE_UPLOAD_ENABLED": "1",
+            "DD_REMOTE_CONFIG_ENABLED": "true",
+            "DD_INTERNAL_RCM_POLL_INTERVAL": "2000",
+            "DD_DEBUGGER_DIAGNOSTICS_INTERVAL": "1",
+            "DD_THIRD_PARTY_EXCLUDES": "org.springframework.samples.petclinic",
```

Comment:
Do we need this? The system tests don't use petclinic, right? Where can I see the code of the java application used for testing?

Reply:
It was your code 😉 
I believe we don't need this. 
Removed here a0d8586ba7d27df1eb1225747690c781d1e26976